recipes/external-systems/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "external-systems" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/external_systems-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
```
- Minor nit: not hard-coding the package name